### PR TITLE
Theme review refactor and cleanup

### PR DESCRIPTION
**Pull request by ravichdev**

### CODE REVIEW

**Review by emeaguiar — APPROVED** [review]

**Comment by emeaguiar** [comment]
Looks good to me, don't know if you want to wait for Derek's review, feel free to merge otherwise

**Comment by ravichdev** [comment]
@derekherman This PR includes changes to bundle webpack concatenated but unminified assets in the zip file, could you review those changes ?

File: inc/admin.php [hunk 1]
```diff
@@ -31,23 +31,10 @@
  * @return void
  */
 function setup() {
-	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\enqueue_admin_assets' );
```
Comment (derekherman) [inline inc/admin.php:34]: So how are assets being enqueued now?

Comment (ravichdev) [reply inc/admin.php:34]: `admin-compiled.css` file contained only few lines of CSS for notifications, I moved those styles to inline https://github.com/xwp/material-design-wp-theme/pull/165/files#diff-93d36a6d7cfdc8b27f5ec6e8d4959420e99295ddf46708ee6fd18489c9ccaf24R92

**Pull request merged**